src/assets/i18n/zh-tw.json

```diff
@@ -94,8 +94,8 @@
     "ifYouNotReceiveEmail": "如果您在10分鐘內仍未收到驗證電子郵件，請單擊下面的按鈕重新發送驗證電子郵件。",
     "emailOrPasswordIsInvalid": "您輸入的電子郵件地址或密碼無效。",
     "tooManyRetries": "您輸入無效的電子郵件或密碼的次數過多。請稍後再試。",
-    "publishingProof": "將拍攝 {{hash}} 發佈至 {{publisherName}}。",
-    "proofPublished": "拍攝 {{hash}} 已發佈到 {{publisherName}}。",
+    "registeringAsset": "正在將拍攝 {{hash}} 發佈至 {{publisherName}}。",
```

Comment:
The zh-tw version doesn't seem to match with the en-us version. (It's not "registering asset in zh-tw")

I would suggest confirm with PO for this translation.